dist/infragram.css

```diff
@@ -97,3 +97,13 @@ div#colorbar-container canvas#colorbar {
 #overlay-img {
   opacity:0.5;
 }
+
+#default_colormap:hover {
+  cursor: pointer;
+  background-color: #e0e0e0;
+}
+
+#stretched_colormap:hover {
+  cursor: pointer;
+  background-color: #e0e0e0;
+}
```

Comment:
In the future, we might decide to add more dropdown items and we would have to come here again to set a style for that item. Instead of setting the style on each list item, can you think of a way to target all the `li` elements in the `ul.dropdown` class? Thanks!